cli/deployment/config.go

```diff
@@ -520,6 +520,12 @@ func newConfig() *codersdk.DeploymentConfig {
 			Flag:    "disable-path-apps",
 			Default: false,
 		},
+		RequestLogLevel: &codersdk.DeploymentConfigField[string]{
+			Name:    "Request Log Level",
+			Usage:   `Set the log level for successful (1xx, 2xx, 3xx, 4xx) request logs. Requests that fail due to a server error (5xx) are always logged at the "warn" level. Valid values are: "info" and "debug".`,
+			Flag:    "request-log-level",
+			Default: "info",
+		},
```

Comment:
Can we just change the default to info instead? People can always just adjust their log level.

Reply:
We got a lot of complaints in v1 about logging way too many things.

Reply:
I'm confused? Do you want me to remove the flag and force it to be info? I don't think customers will want to turn off Info level logging because request logs are spammy

Reply:
@kylecarbs ^

Reply:
I agree with @ammario. I don't find it extremely useful to log all requests by default. Most deployments will have a load-balancer in front that will do similar anyways.